packages/mesa/mesa-vulkan-icd-freedreno.subpackage.sh

```diff
@@ -1,5 +1,6 @@
 TERMUX_SUBPKG_DESCRIPTION="Mesa's Freedreno Vulkan ICD"
 TERMUX_SUBPKG_DEPENDS="vulkan-loader-generic"
```

Comment:
```suggestion
TERMUX_SUBPKG_DEPENDS="libandroid-shmem, libc++, libdrm, libxcb, libxshmfence, libwayland, vulkan-loader-generic, zlib, zstd"
```

Reply:
`TERMUX_SUBPKG_DEPEND_ON_PARENT` is not set to `no` in all of the mesa ICDs. Maybe this variable should be set to `no` and apply these changes because ICDs have no need to depend on mesa in Termux, which provides libraries related to OpenGL.